dakara_server/library/serializers.py

```diff
@@ -67,6 +76,9 @@ class Meta:
 class WorkNoCountSerializer(serializers.ModelSerializer):
     """Work serializer
     """
+    alternative_titles = WorkAlternativeTitleSerializer(
+        many=True,
+        read_only=True)
```

Comment:
It is a bit more readable to indent like this:

```python
    alternative_titles = WorkAlternativeTitleSerializer(many=True,
                                                        read_only=True)
```

Reply:
Didn't think of that !